src/stringobj.c

```diff
@@ -441,13 +445,20 @@ Int             GrowString (
     UInt                len;            /* new physical length             */
     UInt                good;           /* good new physical length        */
 
+    if (need > INT_INTOBJ_MAX)
+        ErrorMayQuit("GrowString: String size too large", 0, 0);
```

Comment:
Perhaps this?
```suggestion
        ErrorMayQuit("GrowString: new string length is too large", 0, 0);
```
Also, why use `ErrorMayQuit` here, but `ErrorQuit` in `NEW_STRING`?